README.md

```diff
@@ -1,488 +1,57 @@
-
-```
-$GOPATH/bin/web-cloudsim
-```
-
-
-## Cloudsim optional Env vars (in the .env file)
-
-- export IGN_CLOUDSIM_SSL_PORT=(default 4431)
-- export IGN_CLOUDSIM_HTTP_PORT=(default 8001)
-
-Also, if using Kubernetes with EC2:
-
-- export KUBEADM_JOIN=kubeadm join IP:PORT --token ...
-- export KUBE_MASTER_IP=the ip of the master node.
-- export IGN_TRANSPORT_IP_INTERFACE=`ethwe` (if using Weave Network in the kubernetes cluster)
-- export AWS_GZ_LOGS_BUCKET=a bucket name to upload gazebo logfiles.
-
-Note: To backup GZ logs into S3 it will also require a `kubernetes secret` called `aws-secrets` in the cluster.
-That secret should at least contain the following `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` entries.
-You can create the secret by running the script `create-secrets.sh` included with the web-cloudsim code.
-These are the contents of that script:
-
-```
-#/bin/bash
-
-# This script is used to create a Kubernetes secret with name 'aws-secrets' based on existing
-# environment variables: AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY.
-# This Secret will be used by the cloudsim server.
-
-kubectl create secret generic aws-secrets --from-literal=aws-access-key-id=${AWS_ACCESS_KEY_ID} --from-literal=aws-secret-access-key=${AWS_SECRET_ACCESS_KEY}
-```
-
-
-Note2: to find the value for the KUBEADM_JOIN env variable, from the kubernetes master, run:
-`sudo kubeadm token create --ttl 0 --print-join-command` and copy the results.
-Read the k8-readme.md for more info.
-
-
-
-# Kubernates
-The cloudsim server will try to connect to a Kubernates master. To do this, it will look
-for the following config file: `<home>/.kube/config`.
-
-
-For more details and tips about kubernetes, see the `k8-readme.md` file.
-
-
-
-# Linter
-
-1. Get the linter (gometalinter)
-
-    ```
-    cd ~/go_ws
-    ```
+[![Format](https://img.shields.io/github/actions/workflow/status/gazebo-web/cloudsim/format.yaml?branch=main&label=Format)](https://github.com/gazebo-web/cloudsim/actions?query=workflow:"Format")
+[![Test](https://img.shields.io/github/actions/workflow/status/gazebo-web/cloudsim/test.yaml?branch=main&label=Test)](https://github.com/gazebo-web/cloudsim/actions?query=workflow:"Test")
+</div>
 
-    ```
-    curl -L https://git.io/vp6lP | sh
-    ```
+## Getting started
 
-1. Run the linter
+The cloudsim library provides multiple components that are used by applications to run simulations in the cloud. These components usually are an abstraction of a third-party service that will be consumed by cloudsim, and they were created in
+order to let the application decide which cloud provider to use.
 
-    ```
-    ./bin/gometalinter $(go list github.com/gazebo-web/cloudsim/...)
-    ```
+A more extensive description of each component and examples on how to create applications will be introduced in a future version of this document.
 
-Note you can create this bash script:
+### Features
```

Comment:
How about we rename this to **Components**?

Reply:
Sure, I went from Components to Features while editing this document, so maybe it makes a lot of sense to use components, wasn't sure about it.